source/frontend/src/hooks/useQueryMapLayersByLocation.ts

```diff
@@ -32,11 +26,19 @@ export interface IMapLayerResults {
  * @returns An object with information + metadata returned by the map layers queried by this hook
  */
 export function useQueryMapLayersByLocation() {
-  const motiRegionService = useLayerQuery(MOTI_REGION_LAYER_URL);
-  const highwaysService = useLayerQuery(HWY_DISTRICT_LAYER_URL);
-  const electoralService = useLayerQuery(ELECTORAL_LAYER_URL);
-  const alrService = useLayerQuery(ALR_LAYER_URL);
-  const firstNationsService = useLayerQuery(INDIAN_RESERVES_LAYER_URL);
+  const {
+    motiRegionLayerUrl,
+    hwyDistrictLayerUrl,
+    electoralLayerUrl,
+    alrLayerUrl,
+    reservesLayerUrl,
+  } = useTenant();
+
+  const motiRegionService = useLayerQuery(motiRegionLayerUrl);
+  const highwaysService = useLayerQuery(hwyDistrictLayerUrl);
+  const electoralService = useLayerQuery(electoralLayerUrl);
+  const alrService = useLayerQuery(alrLayerUrl);
+  const firstNationsService = useLayerQuery(reservesLayerUrl);
```

Comment:
This whole sections is been highlighted by codecov as not covered by tests. I recall we discussed trying to use pr review for adding tests where they are valuable. Are they valuable here?

Reply:
![image](https://user-images.githubusercontent.com/24322224/214990021-af09f334-cd3c-490a-beda-25c08308e03a.png)

Reply:
Test coverage by this story doesn't seem appropriate since this story is enhancing existing functionality that has no tests. Will log test coverage story since that coverage has never existed.